### PR TITLE
Fix static properties

### PR DESCRIPTION
**Pull request by BladeMF**
Index static properties without $

### CODE REVIEW

File: tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ public function testFinder(string $manifest, int $expectedConfirmed, ?int $expec
 
         $locations = iterator_to_array($locations);
 
+        // dump($locations);
```
Comment (dantleech) [inline tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php]: dump

**Comment by dantleech** [comment]
25LOC - not bad :+1: Not sure why the CI didn't start, will have to check that, there is a stray `dump`

**Comment by BladeMF** [comment]
> 25LOC

What is LOC?

**Comment by BladeMF** [comment]
The unit test fails because it depends on WorseReflection working properly.

**Comment by BladeMF** [comment]
https://github.com/phpactor/phpactor/issues/1154

**Comment by dantleech** [comment]
> **What is LOC?**

Lines Of Code

**Comment by BladeMF** [comment]
> > **What is LOC?**
> 
> Lines Of Code

Your code is pretty well structured - I don't need to do much.

**Comment by BladeMF** [comment]
What do we need to do to tell it to use the new version of worse-reflection?

**Comment by dantleech** [comment]
Typically we reference the _next_ version in `composer.json`. So if WR is tagged as `0.1.4` then we require `0.1.5`. Because `minimum-stability: dev` is used it will pick up dev-master (while we use `prefer-stable` to ensure that we don't get dev versions of all packages).

You will need to require `worse-reflection @ 0.4.4` - which will be the version tagged when phpactor is released.

**Comment by BladeMF** [comment]
Currently is `"phpactor/worse-reflection": "^0.4",`. Shouldn't it be 0.5?

**Comment by BladeMF** [comment]
What should we do about 7.4 test failing?

**Comment by dantleech** [comment]
It's the same flakey test I think, I restarted it and it's green now (should increasing the `delay()` time would probably be a very quick fix)

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
         "phpactor/source-code-filesystem": "~0.1.5",
         "phpactor/source-code-filesystem-extension": "^0.1.3",
         "phpactor/text-document": "^1.1.3",
-        "phpactor/worse-reflection": "^0.4",
+        "phpactor/worse-reflection": "^0.4.4",
```
Comment (dantleech) [inline composer.json:23]: :+1:

File: lib/Adapter/Tolerant/Indexer/ClassLikeReferenceIndexer.php [hunk 3]
```diff
@@ -48,10 +50,13 @@ public function beforeParse(Index $index, SplFileInfo $info): void
     public function index(Index $index, SplFileInfo $info, Node $node): void
     {
         assert($node instanceof QualifiedName);
+        
+        $name =
+            ($node->parent->parent instanceof TraitUseClause) ?
```
Comment (dantleech) [inline lib/Adapter/Tolerant/Indexer/ClassLikeReferenceIndexer.php]: don't need the parenthesis here

File: tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php [hunk 4]
```diff
@@ -102,6 +103,54 @@ class Bar extends Foo {}
...
+
+// // File: project/foo.php
+// <?php class Foo { use Bar; };
+
+// // File: project/other.php
+
+// <?php
+// $b = new Foo();
+// $b->func();
+// EOT
```
Comment (dantleech) [inline tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php]: please remove all commented code (in general no TODOs, but especially not commented code).

In general this stuff just doesn't age well.

**Comment by dantleech** [comment]
very nice :+1: just a few comments

File: tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php (hunk 4 above)
Comment (BladeMF) [reply tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php]: Ye, I keep forgetting those. Sorry about that.

Comment (BladeMF) [reply tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php]: Actually I didn't forget it - I meant to leave it as a failing test should someone decide to go after it.

**Comment by BladeMF** [comment]
Rebased on master

**Comment by dantleech** [comment]
Great work :tada:

**Pull request merged**